devops/ci.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       set -eo pipefail
       export ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-6.0/bin/llvm-symbolizer
       export ASAN_OPTIONS="alloc_dealloc_mismatch=0 symbolize=1"
-      ninja check
+      ctest -C $(BuildType) -j $(nproc) --output-on-failure --timeout 400 --interactive-debug-mode 0
```

Comment:
Will this require other users to call the same convoluted line, or does `ninja check` still mean the same thing and you're just making it more configurable in the build?

Reply:
`ninja check` uses the default setting for `ctest`. This does not contain `timeout`s or `output-on-failure`.  I'll do a little bit more, so that `check` has better defaults.